common/test/acceptance/tests/lms/test_lms_instructor_dashboard.py

```diff
@@ -222,14 +199,13 @@ def _create_a_timed_exam_and_attempt(self):
         # Start the proctored exam.
         self.courseware_page.start_timed_exam()
 
-    @flaky  # TODO fix this, see SOL-1183
+    @flaky(max_runs=10, min_passes=10)
```

Comment:
NOTE: I have added the flaky decorator to pass the `max_run` and `min_passes` values to 10 and the test cases are passing now. I'll remove this once reviewed.